ra-optimisation/linear-range-action-rao/src/main/java/com/farao_community/farao/linear_range_action_rao/LinearRangeActionRao.java

```diff
@@ -39,6 +39,17 @@
 public class LinearRangeActionRao implements RaoProvider {
     private static final Logger LOGGER = LoggerFactory.getLogger(LinearRangeActionRao.class);
 
+    public enum SecurityStatus {
+        SECURED,
+        UNSECURED
+    }
+
+    private SecurityStatus securityStatus;
+
+    public void setSecurityStatus(SecurityStatus securityStatus) {
+        this.securityStatus = securityStatus;
+    }
+
```

Comment:
I think you should do that in LinearRangeActionRaoResult instead. So the status is given as a result of the linear rao computation. So you'll have to use `return CompletableFuture.completedFuture((new LinearRangeActionRaoResult(*whatever*)).getExtendable());`

Reply:
OK